### PR TITLE
rework test to avoid multi-epoch gap

### PR DESCRIPTION
**Pull request by jeffwashington**
#### Summary of Changes

Fixes #

### CODE REVIEW

File: runtime/src/bank.rs [hunk 1]
```diff
@@ -8623,8 +8623,8 @@ pub(crate) mod tests {
         let zero_lamports = 0;
         let little_lamports = 1234;
         let large_lamports = 123_456_789;
-        let some_slot = 1000;
-        let rent_collected = 22; // this is a function of 'some_slot'
+        let some_slot = 32; // chosen to cause epoch to be +1
```
Comment (brooksprumo) [inline runtime/src/bank.rs:8626]: Would it be possible to use fields within `genesis_config.epoch_schedule` to compute this value?

File: runtime/src/bank.rs [hunk 2]
```diff
@@ -8731,12 +8731,13 @@ pub(crate) mod tests {
             .accounts_index
             .purge_roots(&zero_lamport_pubkey);
 
-        let some_slot = 1000;
+        let some_slot = 32; // 1 epoch
```
Comment (brooksprumo) [inline runtime/src/bank.rs:8734]: Would it be possible to use fields within `genesis_config.epoch_schedule` to compute this value?

**Review by brooksprumo — COMMENTED** [review]
I think this looks good. I noticed that EpochSchedule has fields that could be used here.

File: runtime/src/bank.rs (hunk 2 above)
Comment (jeffwashington) [reply runtime/src/bank.rs]: used a constant

File: runtime/src/bank.rs (hunk 1 above)
Comment (jeffwashington) [reply runtime/src/bank.rs]: used a constant

**Review by brooksprumo — DISMISSED** [review]

File: runtime/src/bank.rs [hunk 3]
```diff
@@ -8623,8 +8623,8 @@ pub(crate) mod tests {
         let zero_lamports = 0;
         let little_lamports = 1234;
         let large_lamports = 123_456_789;
-        let some_slot = 1000;
-        let rent_collected = 22; // this is a function of 'some_slot'
+        let some_slot = MINIMUM_SLOTS_PER_EPOCH; // chosen to cause epoch to be +1
```
Comment (ryoqun) [inline runtime/src/bank.rs:8626]: iirc, `genesis_config` can return the real `slots_per_epoch` by calling some getter method? (yeah, blame me. i was lazy at the time...)

Comment (ryoqun) [reply runtime/src/bank.rs:8626]: chatted this over slack. i think it's fine as-is.

Comment (jeffwashington) [reply runtime/src/bank.rs:8627]: // genesis_config.epoch_schedule.slots_per_epoch == 432_000 and is unsuitable for this test
unless we turn off warm up or something.

File: runtime/src/bank.rs [hunk 4]
```diff
@@ -8623,8 +8623,9 @@ pub(crate) mod tests {
         let zero_lamports = 0;
         let little_lamports = 1234;
         let large_lamports = 123_456_789;
-        let some_slot = 1000;
-        let rent_collected = 22; // this is a function of 'some_slot'
+        // genesis_config.epoch_schedule.slots_per_epoch == 432_000 and is unsuitable for this test
```
Comment (ryoqun) [inline runtime/src/bank.rs:8626]: :heart:

**Pull request merged**